build_projects/shared-build-targets-utils/Utils/DotNetCli.cs

```diff
@@ -25,6 +25,10 @@ public Command Exec(string command, params string[] args)
         {
             var newArgs = args.ToList();
             newArgs.Insert(0, command);
+            if (command == "restore")
+            {
+                newArgs.Insert(1, "--no-cache");
```

Comment:
Are you sure this is the only place that we do restore?

Reply:
This covers all location other than init-tools.cmd & pack.cmd. A local build with just this change gave me the signed packages we need from CoreClr/CoreFx